conf/cmake/common.cmake

```diff
@@ -38,9 +38,9 @@ endif()
 
 
 if(Config MATCHES "Release")
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -O2 -DNDEBUG")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -O2 -DDEBUG")
```

Comment:
Nah, for Release builds we use -DNDEBUG :-), but this can be addressed later

Reply:
@Ali-Nasrolahi , I merge these changes, but please keep in mind that for release build we use "-DNDEBUG" option.
-DDEBUG or -D_DEBUG is for Debug build
-DNDEBUG or -D_NDEBUG for Release build
Maybe all compiler options I document in one place :) There are not too many.